### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by codedracula4ec**
adds pt-BR translation to the alpha release

### CODE REVIEW

File: po/pt_BR.po
Comment (isneezy) [inline po/pt_BR.po:63]: É de se esperar o mau funcionamento e a inexistência de algumas funcionalidades.

File: po/pt_BR.po
Comment (isneezy) [inline po/pt_BR.po:132]: Define o estilo padrão a ser usado para criar uma nova forma.

**Review by isneezy — CHANGES_REQUESTED** [review]
Hi, here are some few things to hel improve your translation, I'm not sure about the glyphs, but any way thanks. It's nice to see this is attracting people who speak Portuguese like me. :smile:

File: po/pt_BR.po
Comment (codedracula4ec) [reply po/pt_BR.po]: That makes sense. Tanks!

**Comment by Alecaddd** [comment]
@isneezy I'll wait for your approved review on this before merging.
Thank you both for the contribution!

File: po/pt_BR.po
Comment (isneezy) [inline po/pt_BR.po:116]: there is a typo here, it should be `Estilo`

File: po/pt_BR.po
Comment (isneezy) [inline po/pt_BR.po:576]: Gerando pré-visualização, aguarde por favor…

**Review by isneezy — CHANGES_REQUESTED** [review]
I found some more needles

File: po/pt_BR.po
Comment (codedracula4ec) [reply po/pt_BR.po]: fixed on commit df424068db09ebe94af14a03d54bf94d016c2578

**Review by isneezy — APPROVED** [review]

**Pull request merged**
